platform/ui/src/components/IconButton/IconButton.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { useRef } from 'react';
```

Comment:
For naming of components, if you stack the names, they appear next to each other in the file tree. This is a Vue.js styleguide recommendation:

```js
Button
ButtonIcon
ButtonSegmented
```